pinc/unformat.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+function is_format_round($label)
+{
+    return "F" == $label[0];
+}
+
+function remove_formatting($text, $unwrap)
```

Comment:
This probably best belongs in `pinc/DPage.inc` since it manipulates the data for a page.